CHANGELOG.md

```diff
@@ -1,19 +1,21 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Support mirroring of binary files. Thanks to @malice00. ([#2501](https://github.com/realm/realm-js/issues/2501))
+* Performance significantly improved when making a query on the property of a linked table, when the property is indexed. ([]())
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* Fixed an out-of-range bug in Windows builds. In debug mode, the exception `can't dereference out of range vector iterator` will be thrown and the process will terminate. ([realm/realm-object-store#832](https://github.com/realm/realm-object-store/issues/832))
+* A race between extending the file and activity in the encryption layer could lead to crash and corruption. This race has been fixed. The bug may hit on Android, if encryption is in use. It could also affect Linux on file systems where POSIX `prealloc()` is unsupported. ([realm/realm-core#3427](https://github.com/realm/realm-core/pulls/3427))
 
 ### Compatibility
 * Realm Object Server: 3.23.1 or later.
 * APIs are backwards compatible with all previous release of realm in the 2.x.y series.
 * File format: Generates Realms with format v9 (Reads and upgrades all previous formats)
 
 ### Internal
-* None.
+* Updated from Realm Core v5.23.5 to v5.23.6.
```

Comment:
Personally, I'm not a big fan of this format. It is to much clutter compared to the value it provides (IMO), but 🤷‍♂

Reply:
@cmelchior Please see https://github.com/realm/realm-js/pull/2553#discussion_r332880466